### PR TITLE
#38 support windows development

### PR DESCRIPTION
**Pull request by eloisetaylor5693**
#38 Added support for Windows development.

I didn't test every npm script.

Side note:  I think the script should be called `start` instead of `dev`.  eg `yarn start` or `yarn serve:dev`

### CODE REVIEW

File: config/webpack/webpack.client.babel.ts [hunk 1]
```diff
@@ -145,7 +145,7 @@ export default {
     compress: true,
     disableHostCheck: true,
     historyApiFallback: true,
-    host: '0.0.0.0',
+    host: '127.0.0.1',
```
Comment (ivan-aksamentov) [inline config/webpack/webpack.client.babel.ts]: It will break for people who work on remote machines. Why are these necessary?
It works for me with `0.0.0.0` on Windows 7 VM.
If it is really necessary, please make it configurable through an environment variable and add to `.env.example` file

File: package.json [hunk 2]
```diff
@@ -5,10 +5,12 @@
...
   "license": "MIT",
   "scripts": {
+    "predev": "node -e 'console.clear()' && yarn install && yarn dev:clean",
     "dev": "nodemon --config config/nodemon/dev.json",
-    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development babel-node node_modules/.bin/webpack-dev-server --config config/webpack/webpack.client.babel.ts",
+    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development webpack-dev-server --config config/webpack/webpack.client.babel.ts",
     "dev:clean": "rimraf '../.build/development/{*,.*}' 'node_modules/.cache'",
-    "prod": "cross-env NODE_ENV=production BABEL_ENV=production babel-node node_modules/.bin/webpack --config config/webpack/webpack.client.babel.ts",
+    "preprod": "node -e 'console.clear()' && yarn install && yarn prod:clean",
+    "prod": "cross-env NODE_ENV=production BABEL_ENV=production webpack --config config/webpack/webpack.client.babel.ts",
```
Comment (ivan-aksamentov) [inline package.json]: Using pre- hooks only makes things even more confusing in my already super-confusing setup. =)
Could we avoid that? As Microsoft folks explained in the issue thread, it is sufficient to just add spaces around semicolons `;`.
In fact I was also working on it in parallel and it indeed works.

**Comment by ivan-aksamentov** [comment]
Hi @eloisetaylor5693 

I've been working on this slowly in parallel too and I decided to crowdtest it here:
https://github.com/neherlab/covid19_scenarios/issues/38#issuecomment-602241783

and in the end merged my changes, which I found more comprehensive and less breaking.
Here is a summary:
https://github.com/neherlab/covid19_scenarios/compare/28b1468..de97b3b

I will close this for now. But if you still need to adjust something, feel free to jump in to #38 or in the chat or to make another PR

File: config/webpack/webpack.client.babel.ts (hunk 1 above)
Comment (eloisetaylor5693) [reply config/webpack/webpack.client.babel.ts]: On both my windows machines, 0.0.0.0 didn't work.  127.0.0.1 is the standard localhost IP address.

Comment (ivan-aksamentov) [reply config/webpack/webpack.client.babel.ts]: @eloisetaylor5693 Could you please patch and verify it for us?
Add and `.env` variable and set it to `127.0.0.1` by default. Refer to this variable in webpack config.  Those who need remote will change the var to `0.0.0.0`,
Thanks for your investigation!
